CITATION.cff

```diff
@@ -0,0 +1,21 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "van Steen"
+  given-names: "Floyd"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Mirketou"
+  given-names: "Anastasia"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Acquaviva"
+given-names: "Marisa"
+- family-names: "Gonzalez"
+given-names: "Benjamin"
+title: "Collab 1"
+version: 1.0.0.0
+doi: 10.5281/zenodo.1234
+date-released: 2024-10-10
+url: "https://github.com/github-linguist/linguist"
+
```

Comment:
Please add the name of the repo as title.

Reply:
Ready